src/Transformers/TextNormalizer.php

```diff
@@ -43,7 +43,9 @@ public function __construct(bool $uppercase = false)
      */
     public function compatibility() : array
     {
-        return DataType::all();
+        return [
```

Comment:
Actually, the transformer should be compatible with all since it passes through all other types.